tables/clb_access_log/clb_access_log.go

```diff
@@ -97,11 +109,11 @@ func (l *ClbAccessLog) InitialiseFromMap(m map[string]string) error {
 			}
 			l.SentBytes = &sb
 		case "method":
-			method = value
+			l.RequestHTTPMethod = value
 		case "path":
-			path = value
+			l.RequestUrl = value
 		case "http_version":
-			httpVersion = value
+			l.RequestHTTPVersion = value
```

Comment:
for consistency we should name the tokens the same as the fields they're designed to capture?